stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -373,7 +373,7 @@ searchInConformanceCache(const Metadata *type,
     // For generic and resilient types, nondependent conformances
     // are keyed by the nominal type descriptor rather than the
     // metadata, so try that.
-    auto *description = type->getNominalTypeDescriptor().get();
+    auto description = type->getNominalTypeDescriptor().get();
```

Comment:
Unfortunately, `const auto*` does not work. `const auto` does however
![untitled](https://cloud.githubusercontent.com/assets/1275900/22859112/5bc18dea-f104-11e6-8c5f-4cc4c8acc737.png)